### PR TITLE
KVM v12.2.0-2020092800 preview release for helm3

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/11748

### CODE REVIEW

File: kvm/v12.2.0-2020092800/release.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      name: metrics-server
+      version: 1.1.1
+    - name: net-exporter
+      version: 1.9.0
+    - componentVersion: 1.0.1
+      name: node-exporter
+      version: 1.3.0
+  components:
+    - name: app-operator
+      version: 2.3.1
```
Comment (rossf7) [inline kvm/v12.2.0-2020092800/release.yaml:28]:
```suggestion
      version: 2.3.2
```

Can we use the latest release?

**Comment by MarcelMue** [comment]
Why is this 12.2.0 when 12.2.0 already exists? Shouldn't it be `12.3.0-SOMETHING` as a prerelease of 12.3.0

File: kvm/v12.2.0-2020092800/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 12.2.0-2020092800  for KVM :zap:
+
+**If you are upgrading from 12.2.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades all Helm releases managed by Giant Swarm to use [Helm v3.3.0](https://github.com/helm/helm/releases/tag/v3.3.0).
```
Comment (rossf7) [inline kvm/v12.2.0-2020092800/README.md:5]:
```suggestion
This release upgrades all Helm releases managed by Giant Swarm to use [Helm v3.3.4](https://github.com/helm/helm/releases/tag/v3.3.4).
```

File: kvm/v12.2.0-2020092800/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**Note for future 12.x.x releases:**
+
+Please persist this note and the above, until all customers are on AWS v12.2.x and above.
+
+## Change details
+
+### app-operator [v2.3.1](https://github.com/giantswarm/app-operator/blob/master/CHANGELOG.md#231---2020-09-22)
+### chart-operator [v2.3.2](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#232---2020-09-22)
+- Updated Helm to [v3.3.0](https://github.com/helm/helm/releases/tag/v3.3.0).
```
Comment (rossf7) [inline kvm/v12.2.0-2020092800/README.md:29]: Changelogs are for the older version and the Helm version needs to be updated too.

**Review by rossf7 — COMMENTED** [review]
> Why is this 12.2.0 when 12.3.0 already exists? Shouldn't it be 12.3.0-SOMETHING as a prerelease of 12.3.0

@MarcelMue Yes this should be based on 12.3.0. 

@tomahawk28 I found a few other versions that also need updating.

**Pull request merged**